static/styles/zulip.css

```diff
@@ -1440,7 +1440,7 @@ div.focused_table {
 }
 
 #message_edit_topic {
-    height: 20px;
+    height: 27px;
```

Comment:
Yes, that was not requested by the issue its a personal suggestion not well tested
I can certainly bring back the original styles 
@rishig